CHANGELOG.md

```diff
@@ -1,5 +1,37 @@
 # Icinga DB Changelog
 
+## 1.2.0 (2024-04-11)
+
+This release addresses multiple issues related to fault recoveries,
+with a particular focus on retryable database errors that may occur when using Icinga DB with database clusters.
+
+Since there may be a large number of errors that are resolved by retrying after a certain amount of time,
+#698 changed the retry behavior to retry every database-related error for five minutes.
+This helps Icinga DB survive network hiccups or more complicated database situations,
+such as working with a database cluster.
+
+The latter was specifically addressed in #711 for Galera clusters on MySQL or MariaDB by configuring `wsrep_sync_wait` on used database sessions.
+Galera users should refer to the documentation in `doc/03-Configuration.md` for more details.
+
+* Custom Variables: Render large numbers as-is, not using scientific notation. #657
+* Enhance HA "Taking over", "Handing over" logging. #692
+* Retry broken pipe errors (`EPIPE`). #693
+* Retry every database error. #698
+* MySQL/MariaDB: Use strict SQL mode. #699
+* MySQL/MariaDB Galera Cluster: Try setting `wsrep_sync_wait`. #711
+* Don't crash history sync in the absence of Redis. #725
```

Comment:
Maybe this should also get a funny little R, see #736.

Reply:
Added the funny little ᴙ and linked to the trademark document.

<sub><sup>~~As their trademark policy is not RFC 2119 compliant, section 5.a i. could be ignored.~~</sup></sub>